### PR TITLE
MYST-566 Release Docker artifacts 

### PR DESCRIPTION
**Pull request by Waldz**

### CODE REVIEW

File: bin/helpers/docker.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+# Usage:
+# > docker_release_image <version> [tag..]
+#
+# Uploads specific version:
+# > docker_release_image mysteriumnetwork/mysterium-node alpine 0.0.1
+#
+# Uploads several versions
+# > docker_release_image mysteriumnetwork/mysterium-node ubuntu ${VERSION}-ubuntu ubuntu
+docker_release_image () {
+    DOCKER_IMAGE=$1
```
Comment (tadovas) [inline bin/helpers/docker.sh]: use this approach for consuming bash parameters:
DOCKER_IMAGE=$1; shift;
DOCKER_BUILD_TAG=$1; shift;

this way you will keep param consuming and shifting in sync

File: bin/release_github [hunk 2]
```diff
@@ -1,14 +1,14 @@
 #!/bin/bash
```
Comment (tadovas) [inline bin/release_github]: Do we still use this script ?

File: bin/helpers/docker.sh (hunk 1 above)
Comment (Waldz) [reply bin/helpers/docker.sh]: Good one. Fixed

File: .travis.yml [hunk 3]
```diff
@@ -40,21 +39,10 @@ script:
...
-  - BINARY=build/server/mysterium_server_windows_amd64.exe bin/server_package_standalone
-  - BINARY=build/server/mysterium_server_linux_amd64 bin/server_package_standalone
-  - bin/builder_run BINARY=build/server/mysterium_server_linux_amd64 bin/server_package_debian ${VERSION} amd64
-  - bin/client_build_xgo darwin/amd64 windows/amd64 linux/amd64
-  - BINARY=build/client/mysterium_client_darwin_amd64 bin/client_package_standalone
-  - BINARY=build/client/mysterium_client_windows_amd64.exe bin/client_package_standalone
-  - BINARY=build/client/mysterium_client_linux_amd64 bin/client_package_standalone
-  - bin/builder_run BINARY=build/client/mysterium_client_linux_amd64 bin/client_package_debian ${VERSION} amd64
-
+  - bin/package_all $VERSION
```
Comment (tadovas) [inline .travis.yml]: Finally :)

**Pull request merged**